lowatt_enedis/services.py

```diff
@@ -299,8 +303,12 @@ def point_detailed_measures(client, args):
             demande.grandeurPhysique = 'EA'
         elif get_option(args, 'type') == 'COURBE':
             demande.grandeurPhysique = get_option(args, 'courbe_type')
-    demande.soutirage = _boolean(True)
-    demande.injection = _boolean(False)
+    if get_option(args, 'injection'):
+        demande.soutirage = _boolean(False)
+        demande.injection = _boolean(True)
+    else:
+        demande.soutirage = _boolean(True)
+        demande.injection = _boolean(False)
```

Comment:
this could be written more concisely, eg.
```
  injection = get_option(args, 'injection')
  demande.injection = _boolean(injection)
  demande.soutirage = _boolean(not injection)
```

Reply:
Indeed!